_posts/2016-04-08-sprint3_update.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Technical Update (4/8/2016) and Spring Progress Report
+---
+
+The Hydra-in-a-Box technical team's sprint 3 demo -- showing deployment to AWS, multi-tenancy, the configuration UI, and Sufia's new batch upload workflow -- is now available:
+
+<div class="videoWrapper"> <iframe width="1280" height="720" src="https://bluejeans.com/s/9qBr/" frameborder="0" allowfullscreen></iframe></div>
```

Comment:
Ah, I hadn't considered that, but I prefer that. The BlueJeans iframe looks janktastic. Will do.